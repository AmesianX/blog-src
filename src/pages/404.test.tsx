import React from 'react'
import { render, cleanup } from '@testing-library/react'

import NotFoundPage from './404'

describe('page/NotFoundPage', () => {
    afterEach(cleanup)

    it('should render properly', () => {
        const { container } = render(<NotFoundPage/>)
        expect(container).toMatchSnapshot()
    })
})
