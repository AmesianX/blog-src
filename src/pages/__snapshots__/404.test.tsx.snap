// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page/NotFoundPage should render properly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
}

.c1 {
  font-size: 1.3rem;
}

<div>
  <main
    class="c0"
  >
    <h1>
      ¯\\_(ツ)_/¯
    </h1>
    <div
      class="c1"
    >
      404... 요청하신 페이지를 찾을 수 없습니다. 
    </div>
    <a
      href="/"
    >
      홈으로 돌아가기
    </a>
  </main>
</div>
`;
